src/eca/llm_api.clj

```diff
@@ -150,6 +171,27 @@
         :api-key (anthropic-api-key config)}
        callbacks)
 
+      (contains? #{"gemini-2.5-pro"
+                   "gemini-2.5-flash"} model)
+      (llm-providers.google/completion!
+       {:model model
+        :instructions instructions
+        :user-messages user-messages
+        :max-output-tokens max-output-tokens
+        :reason-tokens reason-tokens
+        :reason? (and reason? (:reason? model-config))
+        :past-messages past-messages
+        :tools tools
+        :web-search web-search
+        ;; NOTE: no :api-url here, because Google provider figures it out
+        ;;       because it depends on how we're authenticated
```

Comment:
that's new to me, interesting, I wonder if there is any case user want to customize the url, but it's not a problem right now

Reply:
I did some research (well... Perplexity did most of it ;-)), and outside of running the code within GCP there's no real support for providing custom API urls, apart from custom telemetry endpoints. I'd say that we add a `TODO` and wait for someone to request it.